examples/Hermitian_Phasing.py

```diff
@@ -80,20 +80,21 @@
         logger.info(strm("Rough center is:", rough_center))
         data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
         data.ft("t2")
-        mysgn = select_pathway(data, signal_pathway).C.real.sum("t2").run(np.sign)
-        data *= mysgn
```

Comment:
we should not be sign-flipping our data, because we are not doing a correlation!  (I think I had pointed this out at some point, and it got reverted by accident)

Reply:
this was pulled from the master after you merged in the hermitian phasing PR

Reply:
fixed